### PR TITLE
Update Language Feature Status

### PR DESCRIPTION
**Pull request by 333fred**
Add constant interpolated strings and update records status.

### CODE REVIEW

File: docs/Language Feature Status.md [hunk 1]
```diff
@@ -20,13 +20,14 @@ efforts behind them.
...
 | [Top-level statements](https://github.com/dotnet/csharplang/blob/master/proposals/top-level-statements.md) | [features/SimplePrograms](https://github.com/dotnet/roslyn/tree/features/SimplePrograms)       | [In-Progress](https://github.com/dotnet/roslyn/issues/43563)                                                  | [AlekseyTs](https://github.com/AlekseyTs)                                | [cston](https://github.com/cston), [RikkiGibson](https://github.com/RikkiGibson)                                            | [MadsTorgersen](https://github.com/MadsTorgersen) |
+| [Constant Interpolated Stringsl](https://github.com/dotnet/csharplang/issues/2951) | [features/interpolated-string-constants](https://github.com/dotnet/roslyn/tree/features/interpolated-string-constants) | In Progress | [kevinsun-dev](https://github.com/kevinsun-dev) | [333fred](https://github.com/333fred) | [jaredar](https://github.com/jaredpar), [agocke](https://github.com/agocke) |
```
Comment (jcouv) [inline docs/Language Feature Status.md:30]: Seems like wrong version

File: docs/Language Feature Status.md [hunk 2]
```diff
@@ -35,6 +35,7 @@ efforts behind them.
...
 | [Default in deconstruction](https://github.com/dotnet/roslyn/pull/25562)                           | [decon-default](https://github.com/dotnet/roslyn/tree/features/decon-default)                                  | [Implemented](https://github.com/dotnet/roslyn/issues/25559) | [jcouv](https://github.com/jcouv)                                            | [gafter](https://github.com/gafter)                                                | [jcouv](https://github.com/jcouv)       |
+| [Constant Interpolated Stringsl](https://github.com/dotnet/csharplang/issues/2951) | [features/interpolated-string-constants](https://github.com/dotnet/roslyn/tree/features/interpolated-string-constants) | In Progress | [kevinsun-dev](https://github.com/kevinsun-dev) | [333fred](https://github.com/333fred) | [jaredar](https://github.com/jaredpar), [agocke](https://github.com/agocke) |
```
Comment (cston) [inline docs/Language Feature Status.md:38]: >l [](start = 32, length = 1)

Typo

**Comment by ufcpp** [comment]
How is the status of Top-level statements?

image

**Comment by jaredpar** [comment]
> How is the status of Top-level statements?

Those are already merged into the main branch.

**Comment by hypeartist** [comment]
> > How is the status of Top-level statements?
> 
> Those are already merged into the main branch.

But it has "In-Progress" status in the current [LFS](https://github.com/dotnet/roslyn/blob/master/docs/Language%20Feature%20Status.md)

**Pull request merged**